core/src/test/scala/com.mediative.sparrow/DataFrameReaderTest.scala

```diff
@@ -168,58 +153,12 @@ class DataFrameReaderTest extends FreeSpec with BeforeAndAfterAll {
     }
 
     def testFailure[T: RowConverter: ClassTag](json: Array[String], expected: NonEmptyList[String]) = {
+      val sqlContext = new SQLContext(sc)
       val df = sqlContext.jsonRDD(sc.parallelize(json))
 
       assert(toRDD[T](df) == expected.failure)
     }
 
-    // To get DataFrame#toRDD usage.
-    import com.mediative.sparrow.syntax.df._
-
-    "round-trip an object containing an Int and a String from RDD to DataFrame back to RDD" in {
```

Comment:
@gpoirier so, i'm not ok with removing these tests altogether. i am ok with replacing these tests with better tests. but roundtrip marshalling is important for us to track, because that helps determine how useful or not sparrow is in the real world

please either revert this change, or better yet incorporate your suggested improvements. i want to increase coverage on this front, not reduce it